build.xml

```diff
@@ -326,29 +326,29 @@
         </copy>
     </target>
 
-    <target name="generate-images" description="Generate SVG images of mei files in assets/images/GeneratedImages; depends on docker-mei container image" if="${docker}">
+    <target name="generate-images" description="Generates SVG images of mei files in assets/images/GeneratedImages; depends on docker-mei container image." if="${docker}">
         <exec executable="node" dir="${basedir}/../">
             <arg value="index.js"/>
         </exec>
     </target>
     
-    <target name="generate-images-py">
+    <target name="generate-images-py" description="Generates SVG images of mei files in assets/images/GeneratedImages; depends on python3 with verovio installed.">
```

Comment:
```suggestion
    <target name="generate-images-py" description="Generates SVG images of MEI files in assets/images/GeneratedImages; depends on python3 with Verovio installed.">
```

Reply:
Thanks, updated in parallel description as well.